src/components/scrollManager.js

```diff
@@ -433,39 +443,39 @@ define(["dom", "browser", "layoutManager"], function (dom, browser, layoutManage
     /**
      * Scrolls the document to a given element.
      *
-     * @param {HTMLElement} element target element of scroll task
-     * @param {boolean} [smooth=false] smooth scrolling
+     * @param {HTMLElement} element - target element of scroll task
+     * @param {boolean} [smooth=false] - smooth scrolling
```

Comment:
Will this value always be false?

Reply:
this marks optional + default is _false_